docs/en/ingest-management/processors/processor-add_kubernetes_metadata.asciidoc

```diff
@@ -0,0 +1,229 @@
+[[add_kubernetes_metadata-processor]]
+= Add Kubernetes metadata
+
+++++
+<titleabbrev>add_kubernetes_metadata</titleabbrev>
+++++
+
+// REVIEWERS: Is there really a situation where users need to configure this
```

Comment:
Need input from reviewers on this question.

Reply:
@elastic/obs-cloudnative-monitoring could you all weigh in on this and https://github.com/elastic/observability-docs/pull/500/files#r1036416861

Reply:
There are cases where this processor would be useful yes. I wouldn't consider it as recommended but we should not remove it completely.

Reply:
Agree with Chris